recipes/samtools/0.1.12/meta.yaml

```diff
@@ -0,0 +1,27 @@
+about:
+  home: https://github.com/samtools/samtools
+  license: MIT
+  summary: Tools for dealing with SAM and BAM files
+build:
+  number: 0
+package:
+  name: samtools
+  version: 0.1.12
+requirements:
+  build:
+    - gcc
+    - zlib
+    - ncurses
+  run:
+    - libgcc
+    - zlib
+    - ncurses
+source:
+  fn: samtools-0.1.12.tar.bz2
+  md5: 500ae32fb431de3940e0e955744b9b36
```

Comment:
sha256 preferred

Reply:
The md5 I just copied from sourceforge which might be good enough right?

Reply:
Imho it is better to use a md5 if this is given from upstream then calculating your own checksum.